plugins/inputs/kafka_consumer/kafka_consumer.go

```diff
@@ -133,14 +137,17 @@ func (k *Kafka) receiver() {
 				k.acc.AddError(fmt.Errorf("Kafka Consumer Error: %s\n", err))
 			}
 		case msg := <-k.in:
-			metrics, err := k.parser.Parse(msg.Value)
-			if err != nil {
-				k.acc.AddError(fmt.Errorf("E! Kafka Message Parse Error\nmessage: %s\nerror: %s",
-					string(msg.Value), err.Error()))
-			}
-
-			for _, metric := range metrics {
-				k.acc.AddFields(metric.Name(), metric.Fields(), metric.Tags(), metric.Time())
+			if k.MaxMessageLen != 0 && len(msg.Value) > k.MaxMessageLen {
+				k.acc.AddError(fmt.Errorf("D! Kafka message longer than max_message_len (%d)", k.MaxMessageLen))
```

Comment:
Lets make this a W! level.  AddError should put the plugin name so possibly the Kafka part is redundant, but make sure to add some information about what we are going to do.  Maybe it should be "W! Message longer than max_message_len (%d); dropping message".